.circleci/config.yml

```
@@ -49,33 +51,13 @@ orbs:
               command: |
                 terraform init
           - run:
-              name: Lint Terraform
+              name: Select Workspace
               command: |
-                terraform validate
-      plan_terraform:
-        executor: terraform
-        steps:
-          - checkout
+                terraform workspace select development
```

This can go on the same line like 
```suggestion
                command: terraform workspace select development
```

---

Went with Luke's suggestion of an env var